chain/ethereum/src/chain.rs

```diff
@@ -597,6 +597,13 @@ impl FirehoseMapperTrait<Chain> for FirehoseMapper {
                     number: block.number as i32,
                 },
                 FirehoseCursor::Some(response.cursor.clone()),
+                match block.header {
+                    Some(header) => Some(BlockPtr {
+                        hash: BlockHash::from(header.parent_hash),
+                        number: (block.number as i32) - 1,
```

Comment:
The ideal way of dealing with overflow is to use [checked_sub](https://doc.rust-lang.org/std/primitive.u32.html#method.checked_sub), and if you're sure it's impossible for `block.number` to be 0, then you can use unwrap and comment on why it is safe.

Reply:
fixed, thanks! 
also @maoueh mentionned that the unwrap() on header is also safe (block.header is always populated)